docs/usage/README.md

```diff
@@ -0,0 +1,125 @@
+When a pod is evicted from the node, all corresponding containers are also evicted, together with their logs. 
+This is why the default log storage solution is considerd shot-lived and not sufficient when one wants to operate properly a Kubernetes environment. 
+
+Gardener, as an advanced Kubernetes management solution, follows the general recommendations and offers a cluster level logging solution to ensure proper log storage for all managed Kubernetes resources. 
+The log management is setup when the new cluster is created.
+Log collection is organized using [fluent-bit](https://fluentbit.io). 
+Log storage and search is organized using [Loki](https://grafana.com/oss/loki). 
+Log visualization is available using [Grafana](hhttps://grafana.com/grafana) that is deployed with 
+predefined dashboard and  visualization for every shoot cluster.
+
+
+Kubernetes operators can benefit from different capabilities like accessing the logs for 
+already terminated containers, perform fast and sophisticated search for investigating long 
+lasting problems or recurring problems based on logs from a long period of time. 
+
+Below in this guide, you will find the procedure how to access the log data for your clsuters.
+
+## Procedure
+
+Use this procedure to view the log data of your Kubernetes cluster.
+
+### Accessing Grafana
+1. Open **Gardener dashboard** and navigate to the cluster which log data you want to view
+2. Hover your mouse over cluster name to open and view the details for the cluster. Find the **Logging and Monitoring** section and use the link and credentials to log-in to Grafana.
+
+![Navigate to Logging and Monitoring Tile](images/gardener-dashboard-logging.png)
+
+### Using Grafana
+
+There are two ways for tracking log messages in Grafana.
+
+#### Predefined Dashboards
+The first option is to use the predefined dashboards.
+Go to the **Home** menu and after that you can choose which dashboard to open. The dashboards which contain log visualizations for the different Grafana deployments are:
+
+  * Garden Grafana
+    * Pod Logs
+    * Extensions
+    * Systemd Logs
+  * User Grafana
+    * Kubernetes Control Plane Status
+  * Operator Grafana 
+    * Kubernetes Pods
+    * Kubernetes Control Plane Status
+
+![](images/dashboards.png)
+
+#### Explore menu
+The second option is to use the **Explore** menu.
+The first thing for enabling this option is to authenticate in front of the Grafana UI.
+This happens with the button in the bottom left corner
+![](images/login-button.png)
+
+The default credentials for the Grafana are:
+`username` : `admin`
+`password` : `admin`
+![](images/login-credentials.png)
+
+On successful login you will be asked for changing the default password. Please ***do not*** do that because these credentials will be used from all of the operators so this step ***should be skipped***.
```

Comment:
Avoid "Please" in documentation. Anyway, it's sufficient to warn about the consequence instead of forbidding it, assuming it will be critical. IMO replace with:
```suggestion
On successful login you will be asked for changing the default password.   
**Note:** These credentials are shared among all operators. Changing the default password will affect their access. You can safely skip this step.
```